upgrade/v1beta1/upgrade.go

```diff
@@ -192,15 +223,43 @@ func upgradeNodes(namespace string, lhClient *lhclientset.Clientset) error {
 			return errors.Wrap(err, "failed to copy node Status.DiskStatus")
 		}
 
-		_, err = lhClient.LonghornV1beta2().Nodes(namespace).UpdateStatus(context.TODO(), &new, metav1.UpdateOptions{})
+		obj, err := lhClient.LonghornV1beta2().Nodes(namespace).UpdateStatus(context.TODO(), &new, metav1.UpdateOptions{})
 		if err != nil {
 			return errors.Wrapf(err, "failed to update for %v status %v", old.Kind, old.Name)
 		}
+
+		if err := tagCRLabelCRDAPIversion(obj); err != nil {
+			return errors.Wrapf(err, "failed to add label to %v %v", old.Kind, old.Name)
+		}
+		if _, err := lhClient.LonghornV1beta2().Nodes(namespace).Update(context.TODO(), obj, metav1.UpdateOptions{}); err != nil {
+			return errors.Wrapf(err, "failed to update for %v %v", old.Kind, old.Name)
+		}
 	}
 	logrus.Info("Finished upgrading nodes")
 	return nil
 }
 
+func tagCRLabelCRDAPIversion(obj runtime.Object) error {
+	metadata, err := meta.Accessor(obj)
+	if err != nil {
+		return err
+	}
+
+	labels := metadata.GetLabels()
+	if labels == nil {
+		labels = map[string]string{}
+	}
+
+	labels[types.GetLonghornLabelCRDAPIVersionKey()] = strings.Split(types.CRDAPIVersionV1beta2, "/")[1]
+	metadata.SetLabels(labels)
+
+	return nil
+}
+
+func GetLabelSelectorNotEqualV1beta2APIVersion() string {
```

Comment:
If not exported, suggest using the name staring with a lower case.

Reply:
Okay. Thanks for the reminder.